src/pages/docs/creators/javascript/index.md

```diff
@@ -1,22 +1,63 @@
+
+### JS SDK example projects
+
+Here you have several projects that leverage our libraries:
+
+**tesseract-ocr-golem** - a `golem-js` based library for running Tesseract OCR jobs on the Golem Network.
+
+[GitHub Repo](https://github.com/golemfactory/tesseract-ocr-golem)
+
+**react-image-classifier** - a reference implementation of a React app using Golem underneath.
+
+[GitHub Repo](https://github.com/golemfactory/react-image-classifier)
+
+**music-on-golem** - a reference implementation of a next.js app that uses Golem to generate music. The project contains everything you would find in a typical full-stack application, including a database, design system, authentication, and more.
+
+[GitHub Repo](https://github.com/golemfactory/music-on-golem)
+
+### Golem tools
+
+The creation and running application on the Golem Network may require using of other Golem tools:
+
+**Yagna** - the foundational service that allows users to connect with and interact with the Golem Network.
+
+**Gvmkit-build** - a tool designed to create custom images for virtual machines on provider nodes within the Golem Network. It specifically converts Dockerfiles into Golem-compatible VM images, known as GVMI.
+
+You will find a list of instructions for Golem tools [here](https://docs.golem.network/docs/creators/javascript/examples/tools).
+
+**Golem Registry** - a repository of already built images that can be utilized by Golem Network users [is here](https://registry.golem.network/explore).
+
+## How the documentation is organized
+
+- [Quickstarts](/docs/creators/javascript/quickstarts). If you like to start with something quick and simple for both Node.js and browsers. Just copy the code and follow the instructions.
+
+- [Tutorials](/docs/creators/javascript/tutorials). Start here if you prefer to learn how to write apps on Golem through practical use cases. In the [Running tasks in parallel tutorial](/docs/creators/javascript/tutorials/running-parallel-tasks) you will create an app that recovers passwords using the `hashcat` utility. You will learn how to split the job into multiple smaller tasks and run them in parallel on the Golem Network. You will also create a custom image with the software dedicated to the task and run the app on the network.
+
+- [Guides](/docs/creators/javascript/guides). They are intended for those who prefer to begin by understanding the Golem Network concept and have an overview of the model used. You can also find an explanation of Golem images and instructions on how to create custom ones or how to access the internet from a remote node.
+
+- **JS Task API** agregates [examples](/docs/creators/javascript/examples) and its [API Reference](/docs/golem-js/reference/overview). It will let you explore Task API and find ready-to-run examples with explanations that will help you add these new features to your apps.
```

Comment:
```suggestion
- **JS Task API** agregates introduction to the Task model API, Task API quickstarts, [examples](/docs/creators/javascript/examples) and its [API Reference](/docs/golem-js/reference/overview). It will let you explore Task API and find ready-to-run examples with explanations that will help you add these new features to your apps.
```